static/src/javascripts/es6/projects/common/modules/crosswords/main.js

```diff
@@ -416,6 +417,21 @@ class Crossword extends React.Component {
         this.save();
     }
 
+    onClearSingle () {
+        const cellsInFocus = helpers.cellsForEntry(this.clueInFocus());
+
+        this.setState({
+          grid: helpers.mapGrid(this.state.grid, (cell, gridX, gridY) => {
+              if (_.some(cellsInFocus, cell => cell.x === gridX && cell.y === gridY)) {
```

Comment:
It looks like all our "modern browsers" are ES5 compatible: http://caniuse.com/#feat=es5